discord_slash/client.py

```diff
@@ -54,14 +54,31 @@ def __init__(self,
         else:
             self._discord.add_listener(self.on_socket_response)
             self.has_listener = True
+            default_add_function = self._discord.add_cog
+            def override_add_cog(cog: commands.Cog):
+                default_add_function(cog)
+                self.get_cog_commands(cog)
+            self._discord.add_cog = override_add_cog
+            default_remove_function = self._discord.remove_cog
+            def override_remove_cog(cog: commands.Cog):
+                default_remove_function(cog)
+                self.remove_cog_commands(cog)
+            self._discord.remove_cog = override_remove_cog
 
     def get_cog_commands(self, cog: commands.Cog):
         """
         Gets slash command from :class:`discord.ext.commands.Cog`.
 
+        .. note::
+            Since version ``1.0.X``, this gets called automatically during cog initialization.
```

Comment:
```suggestion
            Since version ``1.0.9``, this gets called automatically during cog initialization.
```
We should put specific version.

Reply:
I left this out on purpose because I was not sure which version to fill it in with. Should've mentioned that.